django_celery_results/migrations/0009_groupresult.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.1.2 on 2020-10-05 17:35
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_celery_results', '0008_chordcounter'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='GroupResult',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('group_id', models.CharField(db_index=True, help_text='Celery ID for the Group that was run', max_length=191, unique=True, verbose_name='Group ID')),
```

Comment:
Make sure to use 
```
max_length=getattr(
                        settings,
                        'DJANGO_CELERY_RESULTS_TASK_ID_MAX_LENGTH',
                        255
                    ),
```
to respect the settings.
see https://github.com/celery/django-celery-results/pull/172